pyproject.toml

```
@@ -48,7 +48,8 @@ dependencies = [
   'tqdm~=4.45',
   'typing-extensions~=4.0;python_version<"3.10"',
   'upf_to_json~=0.9.2',
-  'wrapt~=1.11'
+  'wrapt~=1.11',
+  'chardet'
```

We just pinned most recent version
```suggestion
  'chardet~=5.2.0 ; platform_system == "Windows"'
```
and then you need to run `uv lock --upgrade-package chardet` to put this into the uv.lock file.

---

When you do this we can run the CI